app/models/message.rb

```diff
@@ -0,0 +1,20 @@
+require 'message_broadcast.rb'
+
+class Message < ApplicationRecord
+  belongs_to :conversation
+  belongs_to :user
+
+  validates_presence_of :body
+
+  validate :user_participates_conversation
+
+  after_create_commit { MessageBroadcastJob.perform_later(self) }
```

Comment:
Here use `perform_now` instead of `perform_later` to broadcast the message as soon as posible